profiler/src/Demos/Samples.Computer01/Properties/launchSettings.json

```diff
@@ -26,6 +26,7 @@
         "DD_PROFILING_CPU_ENABLED": "1",
         "DD_PROFILING_EXCEPTION_ENABLED": "1",
         "DD_PROFILING_ALLOCATION_ENABLED": "1",
+        "DD_PROFILING_DEBUG_INFO_ENABLED": "1",
         "DD_INTERNAL_PROFILING_OUTPUT_DIR": "$(PROGRAMDATA)\\Datadog-APM\\Pprof-files\\DotNet",
```

Comment:
As of today, this feature is off by default, because I do not have enough feedback on the library (even though it was implemented by someone from Microsoft, there is no test, the Linux support was not there....)
This feature will not be GA until we are confident, right ?

Reply:
(And I do not consider it as something internal for us, it's a feature like exception profiling which not GA yet)